<launch>
  <!-- Launch an example that sends commands via C++ API. -->

  <arg name="config" default="$(find l2c)/config/panda_simulated_config.yaml"/>

  <node name="servo_server" pkg="moveit_servo" type="servo_server" output="screen" >
    <rosparam command="load" file="$(arg config)"/>
  </node>

  <node name="pose_tracker" pkg="l2c" type="pose_tracker" output="screen">
    <remap from="~target_pose" to="/target_pose" />
    <rosparam command="load" file="$(arg config)" />
    <rosparam command="load" file="$(find l2c)/config/pose_tracking_settings.yaml" />
  </node>

  <node name="trajectory_follower" pkg="l2c" type="trajectory_follower.py" output="screen">
    <rosparam command="load" file="$(find l2c)/config/trajectory_following.yaml" />
  </node>

</launch>